MELA/src/MadMela.cc

```diff
@@ -8,6 +8,8 @@
 #include "TMCFM.hh"
 using MELAStreamHelpers::MELAout;
 using MELAStreamHelpers::MELAerr;
+using namespace madMela;
+std::map<std::pair<TVar::Process, TVar::Production>, MG_process_double>* madMela::updateMap;
```

Comment:
This map serves as a quick way to swap between compute functions (faster than a series of if statements) for different productions/processes